src/imdbtr.js

```diff
@@ -4,15 +4,21 @@ const chalk = require('chalk');
 const figures = require('figures');
 const ora = require('ora');
 const api = require('./api.js');
+const helpers = require('./helpers.js');
 
-const imdbtr = name => {
-  const movie = api(name);
+const imdbtr = params => {
+  if (!params) {
+    return false;
+  }
+
+  const query = helpers.buildQuery(params);
+  const movie = api(query);
 
   if (!movie) {
     return false;
   }
 
-  const spinner = ora(`Searching for ${chalk.yellow(name)}`).start();
+  const spinner = ora(`Searching for ${chalk.yellow(query)}`).start();
```

Comment:
can you improve this information? I've think that "Searching for ${title} from ${year}" is better...

Reply:
Sure. the only problem is if a different set of arguments is used (other than `title` and `year`) so we can look to implement a generic message e.g. "Searching for: key1 = value1 and key2 = value2 and ..." or something similar

Reply:
maybe "Searching for value1, value2", just separated with commas.

Reply:
ok

Reply:
I'll fix later today